app/templates/content/index-crown-hosting.html

```diff
@@ -0,0 +1,100 @@
+{% from 'macros/_breadcrumb.html' import breadcrumb %}
+{% extends "_base_page.html" %}
+
+{% block breadcrumb %}
+  {{ breadcrumb(crumbs) }}
+{% endblock %}
+
+{% block main_content %}
+
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <header class="page-heading">
+          <h1>
+            Physical datacentre space for legacy systems
+          </h1>
+        </header>
+      </div>
+      <div class="column-two-thirds">
+        <p class="service-summary-lede">
+          The Crown Hosting Data Centres framework is available to all central
+          government departments, arm’s length bodies and the wider public sector.
+        </p>
+        <p class="service-summary-lede">
+          Contact <a href="mailto:crownhosting@cabinetoffice.gov.uk">crownhosting@cabinetoffice.gov.uk</a>
+          to discuss datacentre hosting for your legacy systems.
+        </p>
+        <h2 class='summary-item-heading'>
```

Comment:
The spacing here under 'Features' and 'Benefits' doesn't look right (and is different from the existing site):
![screen shot 2015-05-13 at 10 35 55](https://cloud.githubusercontent.com/assets/6525554/7607800/44e1f462-f95c-11e4-9840-4b7ddcf2d181.png)